dev/src/__tests__/courseUtilityLinks.test.tsx

```diff
@@ -0,0 +1,127 @@
+import { Builder, By, Capabilities, until } from 'selenium-webdriver'
+import { isOverlapping } from './testHelpers'
+
+
+describe("Course utility elements should not overlap", () => {
+    //Laptop, desktop, mobile
+    it("Should not overlap on split screen 1920x1080", async () => {
+        const firefox = require('selenium-webdriver/firefox')
+        const service = new firefox.ServiceBuilder('drivers\\geckodriver.exe')
+        const driver = new Builder().forBrowser('firefox').setFirefoxService(service).withCapabilities(Capabilities.firefox()
+            .set("acceptInsecureCerts", true)).build()
+        await driver.get("https://127.0.0.1/dev/#/course/1")
+        await driver.manage().window().setRect({ width: 960, height: 1080 })
```

Comment:
Could these tests be written like this:

```tsx
// `want` can be omitted if we always expect false, regardless of screen resolution
const overlapTests: { width: Number, height: Number, want: boolean }[] = [
    // Insert which resolutions to test here
    { width: 0, height: 0, want: false },
]

overlapTests.forEach(test => {
    it(`Should not overlap on ${test.width}x${test.height}`, async () => {
        ...
        await driver.manage().window().setRect({ width: test.width, height: test.height })
        ...
        expect(overlap).toBe(test.want)
    })
})
```

This way testing for additional resolutions requires you to simply add a line in `overlapTests`, and we reduce code repetition.